packages/ui/src/index.tsx

```diff
@@ -1,7 +1,6 @@
 // components
 export * from './accordion'
 export * from './alert'
-export * from './avatar'
```

Comment:
Can you also remove the folder?

Reply:
It is used in some other components and will be remove after removed those components